jwplatform/constants.py

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
I'm not a fan of generic-purpose files like this. This quickly can become a dumping ground for data that has no relationship. These should probably just go where they're used or in a module that has some logical association to the data (such as upload.py if it's related to configuring how uploads work).

Reply:
This file was removed.